.circleci/config.yml

```diff
@@ -154,7 +154,7 @@ installcrowdsourcingdeps: &installcrowdsourcingdeps
     command: |
       cd ..
       git clone git@github.com:facebookresearch/Mephisto.git Mephisto
-      cd Mephisto; git checkout 7eb85dcbf8bee89a7ac9adc5f23b0f05a78bcb4b -b stable
+      cd Mephisto; git checkout 8f315bfe42ba9643164d3d3d61d7d22609b3ab10 -b stable
```

Comment:
what is this version?

Reply:
That's the most recent version of Mephisto that works with these unit tests. Whenever Mephisto has a new feature that we want to test or a breaking change, we'll have to bump this to keep the tests running

Reply:
Can Mephisto release versions then?

Reply:
We have version tags. Most recent right now is `v0.2.1`.

Reply:
Cool - will use a version tag if available the next time I bump the Mephisto commit